pulsar-client-shaded/pom.xml

```diff
@@ -81,6 +81,8 @@
                   <include>org.apache.pulsar:pulsar-checksum</include>
                   <include>net.jpountz.lz4:lz4</include>
                   <include>com.yahoo.datasketches:sketches-core</include>
+                  <include>org.apache.httpcomponents:httpclient</include>
+                  <include>commons-logging:commons-logging</include>
```

Comment:
Can you also add these in `pulsar-broker-shaded/pom.xml` and `pulsar-client-kafka-compat/pulsar-client-kafka/pom.xml`. I know it's painful, but couldn't find a reliable way to do the shading in the different modules.

Reply:
I see..let me add it.